web/src/beta/lib/reearth-ui/components/Button/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, useCallback, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type ButtonProps = {
+  appearance?: "primary" | "secondary" | "dangerous" | "simple";
+  disabled?: boolean;
+  size?: "normal" | "small";
+  iconButton?: boolean;
+  icon?: string; // TODO: Icon Name, Use Icon Component
+  title?: string;
+  onClick?: () => void;
+};
+
+export const Button: FC<ButtonProps> = ({
+  appearance = "secondary",
+  icon = "",
+  disabled = false,
+  size = "normal",
+  iconButton = false,
+  title = "",
+  onClick,
+}) => {
+  const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
+  const [isActive, setIsActive] = useState<boolean>(false);
+
+  const handleButtonClick = useCallback(() => {
+    if (onClick) onClick();
```

Comment:
I think no need for this function `handleButtonClick` instead you can add `onClick` direct in `StyledButton`